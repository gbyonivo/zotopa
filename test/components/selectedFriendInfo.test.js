import React from 'react';
import { create } from 'react-test-renderer';
import { createStore } from 'redux';
import { Provider } from 'react-redux';
import combineReducer from '../../src/reducers';
import SelectedFriendInfo from '../../src/components/selectedFriendInfo';

describe('SelectedFriendInfo', () => {
  it('should render correctly', () => {
    const actual = create(
      <Provider store={createStore(combineReducer({ transactionReducer: {} }))}>
        <SelectedFriendInfo selectedFriend={{ name: 'ivo', email: '3' }} removeSelection={() => {}} />
      </Provider>
    );
    expect(actual).toMatchSnapshot();
  });
});