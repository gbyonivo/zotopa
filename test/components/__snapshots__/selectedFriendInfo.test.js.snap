// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SelectedFriendInfo should render correctly 1`] = `
<div
  className="sc-bdVaJa ckOkmQ"
  onClick={[Function]}
>
  <label
    className="sc-bwzfXH elRDCf"
  >
    name
  </label>
  ivo
  <label
    className="sc-bwzfXH elRDCf"
  >
    email
  </label>
  3
</div>
`;
